src/osd/OSD.cc

```diff
@@ -8676,6 +8691,7 @@ void OSD::handle_pg_backfill_reserve(OpRequestRef op)
 	m->query_epoch,
 	PG::RemoteReservationRevoked()));
   } else {
+    delete m;
```

Comment:
you can drop these delete calls.. we're about to abort so they just make the code more verbose